frontend/src/app/model/PatientData.ts

```diff
@@ -7,6 +7,8 @@ export interface PatientData {
   personalNumber: string;
   email: string;
   phoneNumber: string;
+  district?: string; // TODO: remove ? when BE is ready
```

Comment:
tohle uz neni potreba, kdyz to sem Lukas dal, ne?

Reply:
@misslecter na tomhle ted pracuje